cogniac/media.py

```diff
@@ -216,14 +216,21 @@ def create(cls,
 
         if filename.startswith('http'):
             args['source_url'] = filename
-        elif fp is None:  # local filename
-            fstat = stat(filename)
+        else:
             if 'media_timestamp' not in args:
                 # set the unspecified media timestamp to the earliest file time we have
                 args['media_timestamp'] = file_creation_time(filename)
-            if stat(filename).st_size > 12 * 1024 * 1024:
+            if fp is None:
+                fsize = stat(filename).st_size
                 # use the multipart interface for large files
-                return CogniacMedia._create_multipart(connection, filename, args)
+            else:
+                fp.seek(0, SEEK_END) # most reliable way of finding file size.
+                fsize = fp.tell()
+                fp.seek(0)
+
+            if fsize > 12 * 1024 * 1024:
+                print("%s using multipart upload" % filename)
```

Comment:
currently sdk is generally nonverbose so prefer to not to add print stuff

Reply:
Amy asks to also remove 
            print(args)
            print(files)
            
can we do it in this pull request for economy of time ?